### PR TITLE
Potuz's correction on PTC and proposer boost and current specs section

### PR DESCRIPTION
**Pull request by thogiti**
## Wiki PR Checklist

- [x] Added a correction suggested by Potuz on PTC section where builder can receive proposer boost per the current ePBS design specs.
- [x] Small grammar changes
- [x] Add a ePBS design specs section.
- [x] Local check on spelling errors and missing words for the wordlist.

### CODE REVIEW

File: docs/wiki/research/PBS/ePBS.md [hunk 1]
```diff
@@ -510,6 +571,206 @@ In principle, if the value at stake in activities or assets secured by Eigenlaye
...
+- As the slot concludes, validators complete several crucial tasks:
+  - **Importing and Validating**: Validators ensure they have imported and validated the inclusion list, the consensus block, all single bit and aggregated attestations, the payload attestations, and the full execution payload.
+  - **Evaluating the Blockchain's New Head**: Based on the data validated, validators make a critical decision on the chain's state.They determine whether the slot results in:
+    - **Full Block**: Both the consensus block and the corresponding execution payload have been successfully imported.
+    - **Empty Block**: The consensus block was imported, but the associated execution payload was not revealed on time.
+    - **Skipped Slot**: No consensus block was imported during the slot, leading to a skipped slot scenario.
+
+
+
+#### Inclusion List Timeline
```
Comment (taxmeifyoucan) [inline docs/wiki/research/PBS/ePBS.md:730]: I assume you plan to finish sections on each of these headers? It's quite a lot of content for already pretty exhaustive doc, I would consider moving it to a separate doc. Also some of the points, like benefits of ePBS are a bit repetitive

**Review by taxmeifyoucan — APPROVED** [review]
Thanks for updating the doc @thogiti! Let me know if you are planning to elaborate more or wanna merge it already

File: docs/wiki/research/PBS/ePBS.md (hunk 1 above)
Comment (thogiti) [reply docs/wiki/research/PBS/ePBS.md:730]: I am mainly focussing on the design specs and summarizing them. Removing lot of non-technical aspects already.

**Comment by thogiti** [comment]
I will try to finish the sections from the design specs tonight and them request you to merge.

**Comment by thogiti** [comment]
@taxmeifyoucan I completed the design specs summary in the `epbs.md`. 

Can you please review when you get a chance? If no major corrections, then we can merge this. 

I will continue working on the based sequencing with preconfs article and anything from tomorrow's gap analysis meeting.

**Pull request merged**